src/gui/plugins/component_inspector/ComponentInspector.qml

```diff
@@ -399,11 +399,222 @@ Rectangle {
+              id: viewSubLidarArea
+              anchors.fill: parent
+              hoverEnabled: true
+              onEntered: lidarSubmenu.open()
+            }
+          }
+     
+          MenuItem {
+            id: magnetometer
+            text: "magnetometer"
+            onTriggered: {
+              ComponentInspector.OnAddEntity(magnetometer.text, "sensor");
+            }
+          }
+        }
+      
+        Menu {
+          id: cameraSubmenu
+          x: addSensorMenu.x - addSensorMenu.width
+          y: addSensorMenu.y + cameraSensorMenu.y
+
+          MenuItem {
+            id: depth
+            text: "Depth"
+             onTriggered: {
+              ComponentInspector.OnAddEntity("depth_camera", "sensor");
+            }
+          }
+          MenuItem {
+            id: logical
+            text: "Logical"
+             onTriggered: {
+              ComponentInspector.OnAddEntity("logical_camera", "sensor");
+            }
+          }
+          MenuItem {
+            id: monocular
+            text: "Monocular"
+             onTriggered: {
+              ComponentInspector.OnAddEntity("camera", "sensor");
+            }
+          }
+          /*MenuItem {
+            id: multicamera
+            text: "Multicamera"
+             onTriggered: {
+              ComponentInspector.OnAddEntity("multicamera", "sensor");
+            }
+          }*/
+          MenuItem {
+            id: rgbd
+            text: "RGBD"
+             onTriggered: {
+              ComponentInspector.OnAddEntity("rgbd_camera", "sensor");
+            }
+          }
+          MenuItem {
+            id: segmentation
+            text: "Segmentation"
+             onTriggered: {
+              ComponentInspector.OnAddEntity("segmentation_camera", "sensor");
+            }
+          }
+          MenuItem {
+            id: thermal
+            text: "Thermal"
+             onTriggered: {
+              ComponentInspector.OnAddEntity("thermal_camera", "sensor");
+            }
+          }
+        }
+
+        Menu {
+          id: lidarSubmenu
+          x: addSensorMenu.x - addSensorMenu.width
+          y: addSensorMenu.y + lidarSensorMenu.y
+
+          /*MenuItem {
+            id: cpuLidar
+            text: "CPU Lidar"
+             onTriggered: {
+              ComponentInspector.OnAddEntity("cpu_lidar", "sensor");
+            }
+          }
+
+          MenuItem {
+            id: cpuRay
+            text: "CPU Ray"
+             onTriggered: {
+              ComponentInspector.OnAddEntity("cpu_ray", "sensor");
+            }
+          }*/
+
+          MenuItem {
+            id: gpuLidar
+            text: "GPU Lidar"
+             onTriggered: {
+              ComponentInspector.OnAddEntity("gpu_lidar", "sensor");
+            }
+          }
+
+          MenuItem {
+            id: gpuRay
+            text: "GPU Ray"
+             onTriggered: {
+              ComponentInspector.OnAddEntity("gpu_ray", "sensor");
```

Comment:
should we just offer `gpu_lidar` **or** `gpu_ray` since they are the same thing?

Reply:
I removed the whole submenu, and just left the GPU lidar in Done in a663abd